dice_ml/model.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, model=None, model_path='', backend=BackEndTypes.Tensorflow1,
             )
 
         self.model_type = model_type
-        if((model is None) & (model_path == '')):
+        if (model is None) and (model_path == ''):

```

Comment:
Probably you could drop these parenthesis as well. See the operator precedence order in the list at the link https://data-flair.training/blogs/python-operator-precedence/#:~:text=Python%20Operator%20Precedence%20%E2%80%93%20Short%20Circuiting%20Python%20always,the%20second%20operand%20only%20until%20it%20is%20needed.